support/troubleshooting/investigating-monitoring-issues.adoc

```diff
@@ -17,14 +17,14 @@ include::modules/monitoring-investigating-why-user-defined-metrics-are-unavailab
 [role="_additional-resources"]
 .Additional resources
 
-* xref:../../monitoring/configuring-the-monitoring-stack.adoc#creating-user-defined-workload-monitoring-configmap_configuring-the-monitoring-stack[Creating a user-defined workload monitoring config map]
-* See xref:../../monitoring/managing-metrics.adoc#specifying-how-a-service-is-monitored_managing-metrics[Specifying how a service is monitored] for details on how to create a service monitor or pod monitor
-* See xref:../../monitoring/managing-metrics.adoc#getting-detailed-information-about-a-target_managing-metrics[Getting detailed information about a metrics target]
+* xref:../../observability/monitoring/configuring-the-monitoring-stack.adoc#creating-user-defined-workload-monitoring-configmap_configuring-the-monitoring-stack[Creating a user-defined workload monitoring config map]
```

Comment:
Change "config map" to "configuration map."

Reply:
Same comment as above re `config map`